packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -6232,36 +6243,38 @@ describe('plugin-meetings', () => {
           });
         });
 
-        it('should reconnectMedia after DX joins after moveTo', async () => {
+        it('should cleanup on moveTo & addMedia after', async () => {
           await meeting.moveTo('resourceId');
 
+          assert.equal(meeting.isMoveToInProgress, true);
+
           await meeting.locusInfo.emitScoped(
             {
               file: 'locus-info',
               function: 'updateSelf',
             },
             'SELF_OBSERVING'
           );
+          
 
-          // beacuse we are calling callback so we need to wait
-
-          assert.called(meeting.cleanupLocalStreams);
-
-          // give queued Promise callbacks a chance to run
+          // Verify that the event handler behaves as expected
+          expect(meeting.statsAnalyzer.stopAnalyzer.calledOnce).to.be.true;
```

Comment:
vast majority of existing tests in plugin-meetings do
`assert.calledOnce(meeting.statsAnalyzer.stopAnalyzer)`
instead of `expect(...calledOnce).to.be.true`
any reason why you're doing it this way?

Reply:
It is just a way of writing tests. I am more used to writing `expects` than `asserts`

I could change everything to assert if required. I saw there are few places in some tests that use `expect` and therefore decided to go with expect for everything